ui/src/components/AlarmsCountBySeverity.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import { groupBy } from 'lodash'
+import { SEVERITY_COLOR } from '@/helpers/constants'
+
+const props = defineProps({
```

Comment:
no harm in assigning this to `const props` here, but it's not necessary unless you want to use `props.relatedAlarms` inside the script tags. In the template, no need to use `props.` to get the value, should just be able to use `relatedAlarms` if you want. Same goes for other files below

Reply:
ah, ok, I thought it was required...

Reply:
its working without it, but it gives me warnings that relatedAlarms are not defined on instance...